src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/config.jelly

```diff
@@ -47,6 +47,10 @@
         <f:textbox default="10"/>
     </f:entry>
 
+    <f:entry title="${%Max number of current connections}" field="maxRequestsPerHostStr">
```

Comment:
I changed the wording a little bit (tried to keep it as short as possible, sacrificing grammar and elegance) and also pushed the corresponding help file which I forgot to `git add` 🤦‍♀️ 🤦‍♂️ , which contains a longer description. Okay like that?